databricks_cli/accounts/cli.py

```diff
@@ -0,0 +1,412 @@
+"""Provide the API methods for Databricks Accounts REST endpoint."""
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from databricks_cli.click_types import AccountIdClickType, CredentialsIdClickType, \
+    StorageConfigIdClickType, NetworkIdClickType, CustomerManagedKeyIdClickType, \
+    WorkspaceIdClickType, JsonClickType
+from databricks_cli.accounts.api import AccountsApi
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, pretty_format, json_cli_base
+from databricks_cli.configure.config import provide_api_client, profile_option, debug_option
+from databricks_cli.version import print_version_callback, version
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Create a credentials object for the AWS IAM role reference.")
+@click.option('--account-id', required=True, type=AccountIdClickType(),
+              help=AccountIdClickType.help)
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST to \
+                    /api/2.0/accounts/{account_id}/credentials.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/accounts/{account_id}/credentials'))
+@debug_option
+@profile_option
```

Comment:
What do these do?

Reply:
Just repeating the pattern from other objects in the CLI. I think @profile_option makes the command open to --profile parameter which would correspond to a particular workspace environment.